config/doctrine/Assignment.orm.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping https://www.doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+    <entity repository-class="App\Repository\AssignmentRepository" name="App\Entity\Assignment" table="assignments">
+        <id name="id" type="integer" column="id">
+            <generator strategy="AUTO"/>
```

Comment:
Shouldn't the strategy be set to `SEQUENCE` or it was left as `AUTO` for "cross-vendor" stuff?

Reply:
It must be set to AUTO, because of cross vendor functionality yes, plus in test environment we use SQLite, which does not have sequences.